packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2588,6 +2596,28 @@ class SemanticsConfiguration {
     _setFlag(SemanticsFlag.namesRoute, value);
   }
 
+  /// Whether the semantics node is a live region.
+  /// 
+  /// A live region indicates that this semantics node will update its 
```

Comment:
This sounds misleading to me. Any semantics node may update its semantics label without being a liveRegion, no? liveRegion just tells a11y systems that a it may be relevant to inform the user about a changed label, no?